src/controllers/check.your.answers.controller.ts

```diff
@@ -37,10 +38,13 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     const overseaEntity: OverseasEntityCreated = await createOverseasEntity(req, req.session as Session, transaction.id as string);
     logger.infoRequest(req, `Overseas Entity Created, ID: ${overseaEntity.id}`);
 
-    await closeTransaction(req, req.session as Session, transaction.id as string, overseaEntity.id);
+    const transactionClosedResponse = await closeTransaction(req, req.session as Session, transaction.id as string, overseaEntity.id);
     logger.infoRequest(req, `Transaction Closed, ID: ${transaction.id}`);
 
-    return res.redirect(config.CONFIRMATION_URL);
+    const redirectPath = await startPaymentsSession(req, req.session as Session, transaction.id as string, overseaEntity.id, transactionClosedResponse);
+    logger.infoRequest(req, `Payments Session ended with, Trans_ID: ${transaction.id}, OE_ID: ${overseaEntity.id}. Redirect to: ${redirectPath}`);
```

Comment:
line 45, I think the 'Payments Session ended' is misleading, as it makes me think that the payment has been taken. I think it would read better if it was 'Payments Session created' which would also match up with the function name startPaymentsSession